nuimo/src/main/kotlin/com/senic/nuimo/NuimoController.kt

```diff
@@ -55,4 +59,5 @@ abstract class BaseNuimoControllerListener: NuimoControllerListener {
     override fun onDisconnect() {}
     override fun onLedMatrixWrite() {}
     override fun onGestureEvent(event: NuimoGestureEvent) {}
+    override fun onBatteryPercentageChanged(batteryPercentage: Int) {}
```

Comment:
We use present tense for event names, such as `onDisconnect ` or `onLedMatrixWrite`. So this one should probably read `onBatteryPercentageChanged `. I decided for this naming pattern as Android SDK tends to use present tense as well, e.g. `onClick` or `onFocusChange`.